### PR TITLE
Running local simulation in the background and notifying progress

### PR DESCRIPTION
**Pull request by kamxgal**

### CODE REVIEW

File: model/Configuration.schema [hunk 1]
```diff
@@ -47,9 +47,9 @@
             "properties" : {
                 "usage": {"type" : "number", "minimum": 0.0},
                 "detection_delay": {"type" : "number", "minimum": 0.0},
-                "testing_delay": {"type" : "number", "minimum": 0.0}
+                "usage_by_household": {"type" : "boolean"}
```
Comment (eMaerthin) [inline model/Configuration.schema:50]: I am not familiar with this change - just to ensure, is this expected `testing_delay` is dropped and `usage_by_household` is added? @tozanski

Comment (tozanski) [reply model/Configuration.schema:50]: yes this is correct, `testing_delay` is not yet used - it was an experiment on another branch and somehow made it into the example JSON

**Comment by tozanski** [comment]
The GUI starts but the RUN does not. There is Julia prompt under `LOG` but nothing happens. It stays in initializing state :( Where does it take Julia from? I believe it is a good idea to bundle the simulation code together with GUI

File: model/SimulationRunner.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+    @pyqtSlot(result=bool)
+    def isRunning(self):
+        return (isinstance(self.__currentState, SimulationRunner.InitState) and \
+            self.__currentState != SimulationRunner.InitState.NONE) or \
+            (isinstance(self.__currentState, SimulationRunner.SimulationState) and \
+            self.__currentState == SimulationRunner.SimulationState.SIMULATION_ONGOING)
+
+    def __createCommand(self):
+        cliname = os.path.basename(self.pathToCLI)
```
Comment (tozanski) [inline model/SimulationRunner.py:63]: why the basename?

Comment (kamxgal) [reply model/SimulationRunner.py:63]: I don't really understand the question. What do you think is wrong with it?

**Pull request merged**